client/settings/settings-manager/style.scss

```diff
@@ -0,0 +1,49 @@
+body.woocommerce_page_wc-settings-checkout,
+body.woocommerce_page_wc-settings-checkout .settings-content,
+body.woocommerce_page_wc-settings-checkout #wpbody-content,
+body.woocommerce_page_wc-settings-checkout #wpcontent {
+	background: #fff !important;
+}
+
+body.woocommerce_page_wc-settings-checkout
+	.woocommerce
+	nav.woo-nav-tab-wrapper {
+	display: none !important;
+}
+
+body.woocommerce_page_wc-settings-checkout .wcpay-settings-layout {
+	margin: 0;
+	max-width: 1432px;
+}
+
+body.woocommerce_page_wc-settings-checkout
+	.css-nsno0f-View-Surface-getBorders-primary-Card-rounded {
```

Comment:
The `css-nsno0f` part of the selector is randomly generated and cannot be used as a static selector.

Reply:
changed in [06d801e](https://github.com/Automattic/woocommerce-payments/pull/10578/commits/06d801ee3d93b9adc1a70a5c51fca680f088c4cd) . thx